sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -22,6 +23,7 @@
  */
 public final class IdentityClientOptions {
     private static final int MAX_RETRY_DEFAULT_LIMIT = 3;
+    private static final String AZURE_REGIONAL_AUTHORITY_NAME = "AZURE_REGIONAL_AUTHORITY_NAME";
```

Comment:
Should `AZURE_REGIONAL_AUTHORITY_NAME` be added to `Configuration`?

Reply:
Are we supposed to add all service environment variables to `azure-core`?